src/Event/EventBundle/Resources/views/Backend/Media/manage.html.twig

```diff
@@ -0,0 +1,57 @@
+{% extends 'EventEventBundle:Backend:layout.html.twig' %}
+
+{% block content %}
+    <section>
+        <div class="page-header">
+            <h3>Media settings</h3>
+            <a href="{{ url('backend_media') }}" >&larr;&nbsp;back</a>
+        </div>
+
+        {{ form_start(form, {'method': 'POST', 'attr': {'id': 'media'}, 'action': path('backend_media_edit', {'id': media.id})}) }}
+        <div class="tab-content">
+            <div class="tab-pane active" id="{{ app.request.locale }}">
+                {{ form_row(form.title) }}
+                {{ form_row(form.filename) }}
+                <div class="uploader-container">
+                    <input class="btn btn-primary" type="button" value="{{ 'Upload file'|trans }}" onclick="showPicker()" />
+                    <input type="hidden" name="file_url" id="file_url"/>
+                </div>
+                {{ form_row(form.description) }}
+                {{ form_row(form.copyrightInfo) }}
+                {{ form_row(form.mediaCredits) }}
+
+            </div>
+
+            {{ form_rest(form) }}
+
+            <input class="btn btn-success" type="submit" value="{{ media.id ? 'Update' : 'Add' }}" />
+        </div>
+        {{ form_end(form) }}
+
+    </section>
+{% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+
+    <script type="text/javascript" src="https://static.filestackapi.com/v3/filestack.js"></script>
+    <script>
+        var client = filestack.init('A3w1aZwZmQBibC09rAcZnz');
```

Comment:
should be taken from parameters.yml at least for now, or better add event settings page where this is can be specified